human_cancer_analysis/nbconverted/1_process_data.py

```diff
@@ -0,0 +1,238 @@
+
+# coding: utf-8
+
+# # Process Powers et. al. data
```

Comment:
Yes the notebook `1_process_data.ipynb` and `nbconverted/1_process_data.py` contain the same content. The `.py` file is just the notebook converted into a script so that you can easily comment in line for these types of PRs.